Sources/TripKit/server/TKWaypointRouter.swift

```diff
@@ -344,26 +306,131 @@ extension TKWaypointRouter {
   
 }
 
-class WaypointParasBuilder {
+// MARK: - Input
+
+extension TKWaypointRouter {
+  
+  struct Input: Encodable {
+    var segments: [Segment]
+    var vehicles: [TKAPI.PrivateVehicle]
+    // var config: TKConfig
```

Comment:
This can be used when we merged this into `main`. I'll create a separate PR for that.

Reply:
noted